### ISSUE TITLE
[docs] Outdated copyright 2017

### ISSUE THREAD

**Issue by vadimkantorov** [pytorch/vision#3476]
```
© Copyright 2017, Torch Contributors.

Built with Sphinx using a theme provided by Read the Docs.
```

**Comment by vfdev-5** [comment]
Maybe, just add `2017-present` to
https://github.com/pytorch/vision/blob/eb00e2ad9da0df409d62033a1d7078572df67fb5/docs/source/conf.py#L86

**Comment by fmassa** [comment]
Fixed in #3513 by @AXE02
